### PR TITLE
Response and Commit to open fields position swaped in LT table

### LINKED ISSUE #547: Response and Commit to Open Appear to be Swapped
### Description:

image

The expanded lead breakdown is what I believe to be correct after comparing the actual PR. 
For some reason, the table fields are showing Commit to Open and Response being swapped compared to tooltip. 

Modifying which columns are visible also is able to make the table appear in an invalid state, 
image
here you can see the lead breakdown under the comment column  
image

I deleted containers and volumes to try again but getting the same result

### PR DESCRIPTION
**Pull request by sahitya-chandra**
resolved #547 

Screenshot from 2024-10-10 19-20-55

### CODE REVIEW

**Comment by jayantbh** [comment]
I think you can remove "good first issue" from the title. 👍

**Comment by jayantbh** [comment]
Could you share a screenshot of you reproducing the issue by yourself and then the fix?

Currently the issue screenshot is from the original reporter, and contains a different selection of columns and data which makes it a slightly less than a 1:1 comparison.

In terms of the code change itself, I think it's sound and needs no further changes.

**Comment by sahitya-chandra** [comment]
@jayantbh This is a current screenshot of the issue

Screenshot from 2024-10-10 21-36-12

**Comment by jayantbh** [comment]
If that's a screenshot of the issue, what does the fix look like?

**Comment by sahitya-chandra** [comment]
The fix is just to swap response and commit to open fields in the table

**Comment by jayantbh** [comment]
I understand, but generally it's a good idea to show visual example of the fix.

For example, what the issue looked like before, and how your change impacted it. :)

It makes it easier for a reviewer to review and approve your PR, and improves trust in your PRs. 🚀 

Given that the response times don't seem to exist in those PRs, you could sync another repo with PR activity, such as Middleware itself.

**Comment by sahitya-chandra** [comment]
Ok sir i will do it right away

**Comment by sahitya-chandra** [comment]
**Previous**
Screenshot from 2024-10-10 22-02-29
Screenshot from 2024-10-10 22-02-37

**After:**
Screenshot from 2024-10-10 22-03-10
Screenshot from 2024-10-10 22-03-18

Sir have a look

**Comment by jayantbh** [comment]
Unrelated... I'm finding it curious about why the response, commit to open, and other times aren't showing up. Is that a different bug? @adnanhashmi09 @samad-yar-khan

**Comment by sahitya-chandra** [comment]
previously i was applying filter in configure options that's why the numbers were not showing

Screenshot from 2024-10-10 22-13-48

**Comment by jayantbh** [comment]
Oh that is strange, because the headers should also hide if you unselect the columns from configure columns.

Is that not what's happening?

**Comment by sahitya-chandra** [comment]
No sir, when i unselect commits or lines their header keeps on showing

**Comment by sahitya-chandra** [comment]
@jayantbh Sir in 2nd commit I have fixed unhiding issue of column when unselected in configure columns.

[Screencast from 2024-10-10 23-00-05.webm](https://github.com/user-attachments/assets/8b7a4120-13b9-40f5-bbf0-a2daa0eaec48)

**Comment by jayantbh** [comment]
Ah, that's more like it. Thanks for sorting that out.
I believe the files aren't properly linted though. Could you run the linter? You'll find instructions in the contributing guide.

**Comment by sahitya-chandra** [comment]
sure sir

**Comment by sahitya-chandra** [comment]
Sir, linting done

**Comment by jayantbh** [comment]
Hey again, so sorry this is one last trivial thing left.
The commands in the contributing guide don't use the --frozen-lockfile flag as they should, resulting in changes to package.json and yarn.lock.

Could you undo changes to those files and push again?

I think the rest is good to go.

**Comment by jayantbh** [comment]
The linter still failed after your last change. Check the above checks.

**Comment by sahitya-chandra** [comment]
- [ ] All checks passed 

- [ ] Linting completed

File: web-server/src/components/PRTable/PullRequestsTableHead.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const PullRequestsTableHead: FC<PullRequestsTableHeadProps> = ({
               <Checkbox
                 checked={allPrsSelected}
                 indeterminate={somePrsSelected}
-                onChange={(e) => {
+                onChange={(e: any) => {
```
[the remark this answers is not in the document]
Comment (jayantbh) [reply web-server/src/components/PRTable/PullRequestsTableHead.tsx:70]: If I didn't miss this earlier, I'd have approved it this time.

[the remark this answers is not in the document]
Comment (sahitya-chandra) [reply web-server/src/components/PRTable/PullRequestsTableHead.tsx]: sir, I have added the correct type now

**Review by jayantbh — APPROVED** [review]

**Comment by jayantbh** [comment]
That's it! Done! Great work @sahitya-chandra! Hopefully your subsequent PRs will go smoother. :)

**Comment by sahitya-chandra** [comment]
Learned a lot this time 


> That's it! Done! Great work @sahitya-chandra! Hopefully your subsequent PRs will go smoother. :)

Learn a lot this time
Thanks @jayantbh sir for your support 
and the PR will surely go very smooth in future

**Pull request merged**